CONTRIBUTING.md

```
@@ -0,0 +1,149 @@
+
+Visit [http://localhost:3000](http://localhost:3000) to view the application frontend and start developing!
+
+## Tinybird Analytics Setup (Optional)
+
+If you are working on the analytics module, to prepare the Tinybird database, follow these steps:
+
+0. We use `pipenv` to manage our Python dependencies. If you don't have it installed, you can install it using the following command:
+   ```sh
+   pkgx pipenv
+   ```
+1. Download the Tinybird CLI from [here](https://www.tinybird.co/docs/cli.html) and install it on your system.
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)